### ISSUE TITLE
Vérifier émetteur SMS

### ISSUE THREAD

**Issue by xTwisten** [RaspbianFrance/raspisms#140]
Bonjour,
Je ne sais pas si cette fonctionnalité existe en tout cas je ne l'ai pas trouvé.
Mais ce qu'il serait peut être intéressant, ça serait de pouvoir vérifier si un numéro à le droit d’exécuter la commande.

Exemple :
- On définit un groupe de confiance, ce groupe contient une liste de numéro qui peuvent exécuter toutes les commandes.
- On peut ajouter un autre numéro de confiance avec "add 0612345678 grp=*". (* pour définir tous les droits sur n'importe quel groupe)

- On créé un groupe "gestion_porte", qui peut exécuter tous les commandes lié au gestion d'une porte.

- Avec un numéro "de confiance" on lui envoi "add 0623456789 grp=gestion_porte". Le numéro indiqué pourra donc exécuter toutes les commandes lié au groupe "gestion_porte".

On pourrait par exemple révoquer un droit à un groupe avec "del 0623456789 grp=gestion_porte" ou même "del 0623456789 grp=*" pour enlever un appareil de confiance.

Enfin si un numéro ne fait pas partie des listes alors j'envoie une alerte aux appareils de confiance.

Je ne sais pas si j'arrive à être claire dans mes explications.

Je vous remercie,

Bonne soirée :)

**Comment by OsaAjani** [comment]
Effectivement, l'idée pourrait être intéressante et n'est pas encore présente pour l'instant. Actuellement la seule restriction sur une commande est la possibilité de rendre le statut admin obligatoire. L'authentification repose uniquement sur un secret (login/password).

Un système d'authentification basé sur le numéro de téléphone me semble tout à fait possible techniquement et serait intéressant. Je verrais pour implémenter l'idée dans une future version.
